core/services/ocr2/plugins/ocr2vrf/fixed_blockhash_provider.go

```diff
@@ -0,0 +1,98 @@
+package ocr2vrf
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/pkg/errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/client"
+	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/ocr2vrf/types"
+)
+
+// fixedBlockhashProvider returns blockhashes with fixed-sized windows relative to current block height
+type fixedBlockhashProvider struct {
+	client client.Client
+	// start block = current head - lookbackBlocks
+	lookbackBlocks uint
```

Comment:
I think you may as well make this (and `batchSize`) `uint64` to reduce the amount of conversions between types you have to do, which just adds code noise.

Reply:
makes sense!